gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/jupiter/handlers/api/v4/DefaultApiReactor.java

```diff
@@ -344,25 +422,43 @@ protected void doStart() throws Exception {
         long startTime = System.currentTimeMillis(); // Get the start Time
 
         // Start resources before
+        resourceLifecycleManager.start();
         policyManager.start();
 
         // Create securityChain once policy manager has been started.
         this.securityChain = new SecurityChain(api.getDefinition(), policyManager, ExecutionPhase.REQUEST);
 
+        if (tracingEnabled) {
+            processorChainHooks.add(new TracingHook("processor-chain"));
+            invokerHooks.add(new TracingHook("invoker"));
+            securityChain.addHooks(new TracingHook("security-plan"));
+        }
+
+        api
+            .getDefinition()
+            .getListeners()
+            .stream()
+            .filter(listener -> listener.getType() == ListenerType.HTTP)
+            .findFirst()
+            .ifPresent(listener -> this.loggingContext = LoggingUtils.getLoggingContext(((HttpListener) listener).getLogging()));
+
+        if (loggingContext != null) {
+            invokerHooks.add(new LoggingHook());
+        }
+
         long endTime = System.currentTimeMillis(); // Get the end Time
         log.debug("API reactor started in {} ms", (endTime - startTime));
 
         this.lifecycleState = Lifecycle.State.STARTED;
+        dumpVirtualHosts();
```

Comment:
What is about?

Reply:
It was there on the original api reactor handler and is there in the SyncApiReactor so I kept it thinking that it is probably useful for some people that want to debug.

Reply:
may we rename it ? because here virtualhost doesn't mean anything especially as we removed this notion with api v4.

Reply:
Yep I renamed it dumpAcceptors() 😁